code/game/turfs/flooring/flooring_decals_industrial.dm

```diff
@@ -28,60 +28,249 @@
 	name = "hatched marking"
 	icon_state = "delivery"
 
+/obj/effect/floor_decal/industrial/hatch/blue
+	name = "cyan hatched marking"
+	color = "#b6efe1"
```

Comment:
should use one of the color defines, like COLOR_BLUE_LIGHT or something (idk I feel `"#b6efe1"` is a bit of a boring color)

also the name should say "blue" to match the type

this applies to all of the /blue decals here

Reply:
It was already like that when i started, see: outlines. It’s not something I did, I’m just copying it because the color gets used. Can fix in a follow up PR.